dist/less/services-view.less

```diff
@@ -127,56 +131,6 @@ services-view,
   .services-items-filter {
     flex: 1 1 100%;
     margin: (@grid-gutter-width / 2) (@grid-gutter-width / 2) (@grid-gutter-width / 4);
-
-    .filter-pf {
```

Comment:
Nope.  All that formatting to get filter results on same line as filter not needed as it was coded that way by default in new pfFilterPanel component.